src/components/ProjectSearch/TextSearch.js

```diff
@@ -153,28 +160,23 @@ export default class TextSearch extends Component {
       }
     });
 
-    return (
-      <span className="line-value">
-        {matches}
-      </span>
-    );
+    return dom.span({ className: "line-value" }, ...matches);
```

Comment:
You get this error when it's an array

```js
    return (
      <span className="line-value">
        {matches}
      </span>
    );
```
<img width="704" alt="screen shot 2017-08-12 at 9 45 03 am" src="https://user-images.githubusercontent.com/254562/29241198-11cae7b8-7f43-11e7-8652-9d9250960494.png">